pkg/rules/regosig/traceerego_test.go

```diff
@@ -347,22 +356,6 @@ func OnEventSpec(t *testing.T, target string, partial bool) {
 			}
 		})
 	}
-
-	t.Run("Should return error when event has unrecognized type", func(t *testing.T) {
```

Comment:
not sure I understand where this piece of code went, if needed

Reply:
It's not needed anymore since there's no "unrecognized type" anymore. If the type is irrelevant for the rule, then evaluation will simply not detect anything. I've added a  new relevant test case with a `nil` finding instead.